src/api/Endpoints/TopLevelFolders.cs

```diff
@@ -98,27 +103,36 @@ public static class TopLevelFolders
 				error = $"{nameof(TopLevelFolderParameters)} is malformed.";
 
 			// Call each of the steps in order and error out if anytyhing fails
-			var storageUri = new Uri($"https://{tlfp.StorageAcount}.dfs.core.windows.net");
+			Result result = null;
+			var storageUri = SasConfiguration.GetStorageUri(account);
 			var fileSystemOperations = new FileSystemOperations(log, new DefaultAzureCredential(), storageUri);
 			var folderOperations = new FolderOperations(log, new DefaultAzureCredential(), storageUri, tlfp.FileSystem);
 
-			Result result = null;
-			result = await folderOperations.CreateNewFolder(tlfp.Folder);
+			// Add folder owner or a larger group to the container ACL
+			var groupACL = Environment.GetEnvironmentVariable("ACL_ROOT_GROUP");

```

Comment:
I get the concept here, but I doubt they're will be one system-wide root group. Perhaps one per storage account?